integration-tests/ccip-tests/actions/ccip_helpers.go

```diff
@@ -1057,6 +1058,10 @@ func (ccipModule *CCIPCommon) DeployContracts(noOfTokens int,
 	return ccipModule.ApproveTokens()
 }
 
+func (ccipModule *CCIPCommon) BlockTime(ctx context.Context) (time.Duration, error) {
```

Comment:
```suggestion
func (ccipModule *CCIPCommon) AvgBlockTime(ctx context.Context) (time.Duration, error) {
```
Might as well keep them named the same